integration-test/common/src/test/java/org/apache/uniffle/test/CoordinatorGrpcTest.java

```diff
@@ -303,20 +297,26 @@ public void shuffleServerHeartbeatTest() throws Exception {
     assertEquals(StorageStatus.NORMAL, infoHead.getStatus());
     assertTrue(node.getTags().contains(Constants.SHUFFLE_SERVER_VERSION));
     assertTrue(scm.getTagToNodes().get(Constants.SHUFFLE_SERVER_VERSION).contains(node));
-    ShuffleServerConf shuffleServerConf = getShuffleServerConf(ServerType.GRPC);
+
+    List<Integer> ports = reserveJettyPorts(1);
+    ShuffleServerConf shuffleServerConf = shuffleServerConfWithoutPort(0, tempDir, ServerType.GRPC);
     shuffleServerConf.set(ShuffleServerConf.STORAGE_MEDIA_PROVIDER_ENV_KEY, "RSS_ENV_KEY");
     String baseDir = shuffleServerConf.get(ShuffleServerConf.RSS_STORAGE_BASE_PATH).get(0);
+    shuffleServerConf.setString(ShuffleServerConf.RSS_STORAGE_BASE_PATH.key(), baseDir);
     String storageTypeJsonSource = String.format("{\"%s\": \"ssd\"}", baseDir);
     withEnvironmentVariables("RSS_ENV_KEY", storageTypeJsonSource)
         .execute(
             () -> {
               // set this server's tag to ssd
               shuffleServerConf.set(ShuffleServerConf.TAGS, Lists.newArrayList("SSD"));
+              shuffleServerConf.setString("rss.coordinator.quorum", getQuorum());
+              shuffleServerConf.setInteger(RssBaseConf.RPC_SERVER_PORT, 0);
+              shuffleServerConf.setInteger(RssBaseConf.JETTY_HTTP_PORT, ports.get(0));
               ShuffleServer ss = new ShuffleServer(shuffleServerConf);
               ss.start();
               grpcShuffleServers.set(0, ss);
             });
-    Thread.sleep(3000);
+    Thread.sleep(2000);
```

Comment:
What is the reason for reducing the sleep time here?

Reply:
Keep the origin